Content.Server/StationEvents/Events/StationEvent.cs

```diff
@@ -112,6 +123,7 @@ public virtual void Startup()
         {
             Started = true;
             Occurrences += 1;
+            LastRun = IoCManager.Resolve<GameTicker>().RoundDuration();
```

Comment:
```suggestion
            LastRun = IoCManager.Resolve<IEntitySystemManager>().GetEntitySystem<GameTicker>().RoundDuration();
```

Reply:
Can you use `EntitySystem.Get<GameTicker>()` so it matches the other code in that file?